certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -153,6 +151,8 @@ func (p *DeployableByOlmCheck) operatorMetadata(bundleRef certification.ImageRef
 		PackageName:      packageName,
 		App:              packageName,
 		InstallNamespace: packageName,
```

Comment:
Since we are now parsing the `CSV` does it make sense to use the value for `InstalledNamespace` from whatever is listed in `operatorframework.io/suggested-namespace` if this key is present, or is that overkill? @tonytcampbell Any thoughts on this?

Reply:
I think this may be something to consider for a future enhancement. We can create a GH issue to track that as well.

Reply:
Thanks for the input, will create an issue for this and resolve this comment.